### ISSUE TITLE
ModelEngine API not supported

### ISSUE THREAD

**Issue by TenReeez** [MagmaGuy/EliteMobs#98]
[02:39:47 WARN]: [EliteMobs] Model Engine API version is not supported. Currently Elitemobs can only support Model Engine R2.2.0, documentation for other versions doesn't exist.

Model engine is now 2.3.2 Will you be updating? 
If not, is this error cause for concern besides console spam?

Paper: 1.18.1
EM: 7.3.12
Model Engine: R2.3.2

**Comment by MagmaGuy** [comment]
For me to update to 2.3.2 the author has to release the API for it. He hasn't. The last API release was 2.3.1, which was only released because I went and asked the author directly. That API release is bugged and will simply not work. I have notified the author about that on the 16th of January and it is still not working.

As for the errors, I have no clue. Best I can tell the API has completely changed at least twice if not three times between 2.2.0 and 2.3.2 in a way that completely broke backwards compatibility. Originally 2.3.1 was so different that it simply did not work, but I got reports from people saying that 2.3.2 was working for them at least to some degree, which I guess means it completely changed again for that version.

Until the Model Engine author actually releases the API for the current version - and does so in a way that isn't bugged - it's not that I don't want to update, it's that it simply is not possible to update.
